lib/ruby_lsp/requests/semantic_highlighting.rb

```diff
@@ -24,8 +24,6 @@ class SemanticHighlighting < Request
       extend T::Sig
       extend T::Generic
 
-      ResponseType = type_member { { fixed: T::Array[Listeners::SemanticHighlighting::SemanticToken] } }
-
```

Comment:
The `extend T::Generic` is no longer necessary if we remove the declaration of the generic type